tests/helpers.py

```diff
@@ -1345,3 +1345,7 @@ def get_memory_leak_median_value():
             raise UnexpectedBehaviour
         median_dict[f"{worker}"] = statistics.median(memory_leak_data)
     return median_dict
+
+
+def get_osd_count():
```

Comment:
what do you think about having a function that returns the count of pods according to the label selector?
Also, this function should be moved to pod.py
Add add docstring.

Reply:
pod.py has get_osd_pods(), This func would return the length of osd count

Reply:
I meant maybe we can have a function that returns the number of pods for a given label selector

Reply:
got it, thanks